spec/epochs_spec.lua

```diff
@@ -937,4 +938,30 @@ describe("epochs", function()
 			assert.are.equal(currentTimestamp + 1, _G.NextEpochsPruneTimestamp)
 		end)
 	end)
+
+	describe("getRewardRateForEpoch", function()
+		it("returns 0.05% for the first 365 epochs (one year)", function()
+			assert.are.equal(0.0005, epochs.getRewardRateForEpoch(0))
+			assert.are.equal(0.0005, epochs.getRewardRateForEpoch(1))
+			assert.are.equal(0.0005, epochs.getRewardRateForEpoch(364))
+			assert.are.equal(0.0005, epochs.getRewardRateForEpoch(365))
```

Comment:
If we're zero-indexing epochs, shouldn't the final epoch at 0.05% be 364?

Reply:
I think the first epoch is EpochIndex 1. so the test is confusing to check for 0, removed that